test/python/algorithms/eigensolvers/test_vqd.py

```diff
@@ -116,7 +115,16 @@ def test_full_spectrum(self):
             result.eigenvalues.real, self.h2_energy_excited, decimal=2
         )
 
-    @data(H2_PAULI, H2_OP)
+    def test_betas_autoeval(self):
```

Comment:
Is there a way we could test the autoevaluation itself? This here would also fail if something else in the algorithm is going wrong 🙂 What about e.g. checking the log?

Reply:
Yeah... I thought about maybe adding the betas to the metadata to be able to check via callback, it's either that or overwriting `self.betas` with the autoevaluated value.